IOPool/Output/src/PoolOutputModule.cc

```diff
@@ -476,7 +476,7 @@ namespace edm {
             "Set ROOT auto flush stored data size (in bytes) for event TTree. The value sets how large the compressed "
             "buffer is allowed to get. The uncompressed buffer can be quite a bit larger than this depending on the "
             "average compression ratio. The value of -1 just uses ROOT's default value. The value of 0 turns off this "
-            "feature.");
```

Comment:
is "-1" still special? (or the ROOT default is to flush after each event?)

Reply:
-1 is still special as if the parameter is set to -1 we do not call `TTree::SetAutoFlush`.